docs_src/microservices/configuration/CommonConfiguration.md

```diff
@@ -70,6 +70,7 @@ The tables in each of the tabs below document configuration properties that are
     |MaxResultCount|1024*|Read data limit per invocation. *Default value is for core/support services. Application and Device services do not implement this setting. |
     |MaxRequestSize|0|Defines the maximum size of http request body in kilbytes. 0 represents default to system max.|
     |RequestTimeout         |5s                          | Specifies a timeout duration for handling requests |
+    |EnableNameFieldEscape|false|The name field escape could allow the system to use special or Chinese characters in the different name fields, including device, profile, and so on.  If the EnableNameFieldEscape is false, some special characters might cause system error.|
```

Comment:
Add the following statment:
`If EnableNameFieldEscape is true, the client of event or command message bus API clients have to escape the name to subscribe the topics.`
Also, add an example here.  The event topic with "-" in the device name and profile name

Reply:
Updated the description.